src/styles/card.scss

```diff
@@ -0,0 +1,96 @@
+@import "variables";
+
+.card {
+  width: 167px;
```

Comment:
Set width to 200px. Add box-sizing border box so paddings will be counted in that value of 200px